formbricks/formbricks
# docs: Add Google OAuth Integration Guide

## What does this PR do?

This PR adds comprehensive documentation for integrating Google OAuth authentication with the Formbricks instance. It guides the users through the steps required to set up a Google Cloud Platform project, configure OAuth 2.0, and update the Docker environment variables for the Formbricks instance. This documentation is crucial for enabling a secure and streamlined user login experience via Google accounts.

Fixes # (issue) [Replace with the issue number if there is a related issue]

## Type of change


## How should this be tested?

1. Review the added documentation for clarity, accuracy, and completeness.
2. Follow the documentation instructions to set up Google OAuth with a Formbricks instance and ensure the steps are clear and reproducible.

## Checklist

### Required

- [x] First PR at Formbricks? [Please sign the CLA!](https://formbricks.com/clmyhzfrymr4ko00hycsg1tvx)

### Appreciated
